includes/class-wcs-att-cart.php

```diff
@@ -129,6 +132,11 @@ public static function get_cart_subscription_schemes() {
 			if ( WCS_ATT_Product::is_subscription_product_type( $cart_item[ 'data' ] ) ) {
 				return false;
 			}
+
+			// Renewing/Resubscribing?
+			if ( isset( $cart_item[ 'subscription_renewal' ] ) || isset( $cart_item[ 'subscription_initial_payment' ] ) || isset( $cart_item[ 'subscription_resubscribe' ] ) ) {
+				return false;
```

Comment:
@thenbrent this will prevent cart-level subscription schemes from being added and must be removed from here in order to allow these types of renewals/resubs. Not sure if this makes sense?

I need to fix this on the "displaying" side of things -- e.g. "when resubscribing/renewing, append the cart-level (global) schemes to objects but don't display any options in the cart page". Will see if I can patch it today 👍